Avia: fixed moscow123-bug

TsoiAlex82 opened the pull request:
Avia: bug with "123москва123" fixed. After get_answer data processes in a separate function(extract_data_from_answer) before rendering.

zrishat commented on travelru/waysearch/views.py:
```diff
@@ -83,7 +84,33 @@ def from_city_to_code(city):
             airport = city.split()[-1].strip("()")
             if iata_tag['name'] == airport:
                 return iata_tag['code']
+            else:
```
Тут надо переделать на if iata_tag['name'] == 'все аэропорты'

zrishat commented:
надо будет исправить конфиликты, поменялись формы в прошлом PR @TsoiAlex82

TsoiAlex82 committed:
Avia: fixed moscow123-bug
Avia: flights data from get_request_answer processes in a separate function before rendering
